memcached.c

```diff
@@ -6419,78 +6419,91 @@ static void usage(void) {
            "                           binary protocol. speeds up all sets.\n"
            "   - no_hashexpand:       disables hash table expansion (dangerous)\n"
            "   - modern:              enables options which will be default in future.\n"
-           "             currently: nothing\n"
-           "   - no_modern:           uses defaults of previous major version (1.4.x)\n"
+           "                          currently: nothing\n"
+           "   - no_modern:           uses defaults of previous major version (1.4.x)\n");
 #ifdef HAVE_DROP_PRIVILEGES
-           "   - drop_privileges:     enable dropping extra syscall privileges\n"
+    printf("   - drop_privileges:     enable dropping extra syscall privileges\n"
            "   - no_drop_privileges:  disable drop_privileges in case it causes issues with\n"
-           "                          some customisation.\n"
+           "                          some customisation.\n");
 #ifdef MEMCACHED_DEBUG
-           "   - relaxed_privileges: Running tests requires extra privileges.\n"
+    printf("   - relaxed_privileges: running tests requires extra privileges.\n");
 #endif
 #endif
 #ifdef EXTSTORE
-           "   - ext_path:            file to write to for external storage.\n"
+    printf("   - ext_path:            file to write to for external storage.\n"
            "                          ie: ext_path=/mnt/d1/extstore:1G\n"
-           "   - ext_page_size:       size in megabytes of storage pages.\n"
+           "   - ext_page_size:       size in megabytes of storage pages. (default: 64 MB)\n"
            "   - ext_wbuf_size:       size in megabytes of page write buffers.\n"
-           "   - ext_threads:         number of IO threads to run.\n"
-           "   - ext_item_size:       store items larger than this (bytes)\n"
-           "   - ext_item_age:        store items idle at least this long\n"
-           "   - ext_low_ttl:         consider TTLs lower than this specially\n"
+           "                          (default: 4 MB)\n"
+           "   - ext_threads:         number of IO threads to run. (default: 1)\n"
+           "   - ext_item_size:       store items larger than this (bytes; default: 512)\n"
+           "   - ext_item_age:        store items idle at least this long (default: 4294967295)\n"
+           "   - ext_low_ttl:         consider TTLs lower than this specially (default: 0)\n"
            "   - ext_drop_unread:     don't re-write unread values during compaction\n"
-           "   - ext_recache_rate:    recache an item every N accesses\n"
+           "                          (default: disabled)\n"
+           "   - ext_recache_rate:    recache an item every N accesses (default: 2000)\n"
            "   - ext_compact_under:   compact when fewer than this many free pages\n"
+           "                          (default: total storage / (ext_page_size * 4) )\n"
```

Comment:
Is it alright to be descriptive for a default like this? There's a danger that this could go out of sync with the actual logic.

Reply:
I think so, though maybe "1/4th of the assigned storage" is more clear.